packages/toolpad-studio/src/server/functionsRuntime.ts

```diff
@@ -1,15 +1,17 @@
 import * as path from 'path';
+import * as url from 'url';
 import * as fs from 'fs/promises';
 import { TOOLPAD_DATA_PROVIDER_MARKER, ToolpadDataProvider } from '@toolpad/studio-runtime/server';
 import * as z from 'zod';
 import { fromZodError } from 'zod-validation-error';
 import * as crypto from 'crypto';
 
 async function loadExports(filePath: string): Promise<Map<string, unknown>> {
-  const fullPath = path.resolve(filePath);
-  const content = await fs.readFile(fullPath, 'utf-8');
+  const importFileUrl = url.pathToFileURL(path.resolve(filePath));
```

Comment:
In case vscode doesn't suffice? 😄 

![Screenshot 2024-04-18 at 09 27 30](https://github.com/mui/mui-toolpad/assets/2109932/1248a22c-9f71-46c4-8339-2ea3b38f276a)

Reply:
😄 I thought it deserves special placement because of the bug. Perhaps we can link to the issue

Reply:
👍 I've added a comment